recipes-iec61850/iec61850-open-server/iec61850-open-server.bb

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2021, Alliander (http://www.alliander.com)
+# SPDX-License-Identifier: Apache-2.0
+
+LICENSE = "GPL-3.0"
+LIC_FILES_CHKSUM = "file://COPYING;md5=d32239bcb673463ab874e80d47fae504"
+
+SRC_URI = "git://github.com/robidev/iec61850_open_server.git;protocol=https;branch=master"
+
+#PV = "1.0+git${SRCPV}" # TODO check if this should be included
```

Comment:
As described in Yocto documentation:

 The version is normally extracted from the recipe filename. For example, if the recipe is named.

Reference: https://www.yoctoproject.org/docs/latest/ref-manual/ref-manual.html#var-PV
I would set the version in recipe name, for instance:
iec61850-open-server_git.bb .

Do you plan to use version in iec61850_open_server.git

Reply:
I probably should, as not to break anything in future builds, so yes, I will create a 'release' version.

Reply:
Made a release v0.9-alpha(tagged as pre-release), as it is functional/usable, but not production ready